### PR TITLE
SAK-48121 portal: Add popover for site description

### PR DESCRIPTION
**Pull request by adrianfish**
https://sakaiproject.atlassian.net/browse/SAK-48121

### CODE REVIEW

**Comment by adrianfish** [comment]
Screenshot 2022-11-25 at 18 25 26
Screenshot 2022-11-25 at 18 25 16

File: portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/includeSiteNav.vm [hunk 1]
```diff
@@ -17,27 +17,37 @@
...
-            <button class="btn btn-nav btn-site rounded-end text-start border-0 me-1 d-flex flex-grow-1 #if (!$isCurrentSite)collapsed#end"
-                    data-bs-toggle="collapse"
-                    data-bs-target="#site-${index}-pages"
-                    aria-expanded="${isExpanded}"
-                    aria-controls="site-${index}-pages">
-                <i class="bi bi-chevron-right" aria-hidden="true"></i>
-                <span>${site.title}</span>
-            </button>
+        <div class="site-list-item-head d-flex align-items-center pe-2">
+            <div>
```
Comment (kunaljaykam) [inline portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/includeSiteNav.vm:21]: Is this 'div' required?

Comment (adrianfish) [reply portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/includeSiteNav.vm]: No. None of those divs I added were needed actually. Well spotted, thanks.

Comment (kunaljaykam) [reply portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/includeSiteNav.vm]: No problem :)

**Review by kunaljaykam — APPROVED** [review]

**Pull request merged**